libandroid-navigation/build.gradle

```diff
@@ -42,8 +42,17 @@ android {
 
 dependencies {
   // Mapbox Android Services
-  compile dependenciesList.mapboxServices
+  compile 'com.mapbox.mapboxsdk:mapbox-android-telemetry:2.2.6'
```

Comment:
@cammace why hardcoding the dependencies? Right, now this wouldn't be necessary because `'com.mapbox.mapboxsdk:mapbox-android-services:2.2.6@aar'` is including `mapbox-android-telemetry`.
I imagine this is in preparation for the upcoming version of MAS, right? BTW, we should keep including the definitions of the dependencies in `dependenciesList`.